2.4/examples/replica/mongodb-clustered.json

```diff
@@ -94,7 +94,13 @@
                 "command": [
                   "run-mongod", "initiate"
                 ],
-                "containerName": "mongodb"
+                "containerName": "mongodb",
+                "env": [
+                  {
+                    "name": "MONGODB_NODES_TO_WAIT",
```

Comment:
I prefer shorter, but I will leave decision on you ;-) MONGODB_ is good for namespacing ;-)